src/Xpress.jl

```diff
@@ -92,7 +92,15 @@ module Xpress
 
         # some lics require special check
         # -------------------------------
-        userlic()
+        xpauth_path = ""
+        if is_unix()
+          if haskey(ENV, "XPRESSDIR")
+            xpauth_path = joinpath(ENV["XPRESSDIR"], "bin")
+          else
+            xpauth_path = joinpath(dirname(dirname(xprs)), "bin")
+          end
+        end
+        userlic(xpauth_path=xpauth_path)

```

Comment:
Sorry, I meant that Xpress looks for the **xpauth.xpr** file in `XPAUTH_PATH` before looking in `XPRESSDIR`, the **libxprs.dylib** should always be in `XPRESSDIR`